mix.exs

```diff
@@ -52,7 +52,7 @@ defmodule Phoenix.LiveDashboard.MixProject do
       {:telemetry_metrics, "~> 0.6 or ~> 1.0"},
       {:ecto_psql_extras, "~> 0.7", optional: true},
       {:ecto_mysql_extras, "~> 0.5", optional: true},
-      {:ecto_sqlite3_extras, "~> 1.1.7", optional: true},
+      {:ecto_sqlite3_extras, "~> 1.2.0", optional: true},
```

Comment:
```suggestion
      {:ecto_sqlite3_extras, "~> 1.1.7 or ~> 1.2.0", optional: true},
```

Reply:
Updated. Is this preferred so downstream projects can use either? Should I update the lock file as well?

Reply:
Yes, this loosens the downstream requirement. I think the lock file is fine, thanks! :)